apps/10address-book-ajax/script.js

```diff
@@ -1,5 +1,41 @@
 'use strict';
 
 $(document).ready(function() {
-    // You code here
-});
\ No newline at end of file
+  $.ajax('https://reqres-api.herokuapp.com/api/users', {
+
+    success: function(response) {
+      response.forEach(function(user){
+        var str =
+          '<tr>' +
+            '<td>' + user.id + '</td>' +
+            '<td>' + user.first_name + '</td>' +
+            '<td>' + user.last_name + '</td>' +
+            '<td>' + '<a href="#"' + 'data-id=' + user.id + '>' + 'view' + '</a>' + '</td>' +
+          '</tr>';
+
+        var $tableRow = $(str);
+
+        $('tbody').append($tableRow);
+
+        $('a', $tableRow).on('click', function(){
```

Comment:
You forgot to pass in 'event' as a parameter here, thus event.preventDefault(); will run as undefined. Ex:

$('a').on('click', function(event){
        event.preventDefault();
        ...
});